.github/workflows/gradle-check.yml

```diff
@@ -1,5 +1,6 @@
 name: Gradle Check (Jenkins)
 on:
+  push:
```

Comment:
Do we need to add push event back, previously we discussed this is not needed ?

Reply:
This is for branches not for pull request if you specify the conditions of PR events, this wont overlap.